README.md

```diff
@@ -6,6 +6,15 @@ English | [简体中文](./README-zh_CN.md)
 
 Please be aware that the demos may exhibit significant accessibility issues, such as problems with keyboard navigation, speech synthesis, and progressive enhancement or degradation.
 
+
+## slided button added 
```

Comment:
Dont add this here, before adding please look through the whole code, add it down where it belongs to.